public/css/nexthome.css

```diff
@@ -1,530 +1,660 @@
+    margin-bottom: 30px;
+}
+.blocks-holder {
+    display: grid;
+    grid-template-columns: repeat(auto-fill,minmax(250px, 1fr));
+    grid-gap: 20px;
+}
+.block {
+    background: white;
+    overflow: hidden;
+    transition: 0.2s ease-in-out;
+    border: 1px solid #c4cbd2;
+    position: relative;
+}
+.block-info {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding: .2em .5em;
     background: white;
-    padding:15px 35px;
+    transition: 0.2s ease-in-out;
+    overflow: hidden;
+    font-size: 14px;
+    border-bottom: 1px solid #c4cbd2;
+}
+.block-info > * {
+    flex-basis: 0;
+    flex-grow: 1;
+    text-align: center;
+}
+.block-info a.color-code {
     font-family: "Source Sans Pro", sans-serif;
-    border-radius: 4px;
-    margin-bottom: 20px;
-    margin-right: 35px;}
-    .networkStatsWrapper .statCard img {
-      margin-top: 4px;
-      margin-bottom: 12px;
-      margin-right: 12px;
-      float: left; }
-    .networkStatsWrapper .statCard p.statcard-title {
-      font-family: "Source Sans Pro", sans-serif;
-      color: #17173C;
-      margin-bottom: 0px;
-      font-size: 14px; }
-    .networkStatsWrapper .statCard .heading {
-      color: #8997A5;
-      font-size: 12px; }
-    .networkStatsWrapper .statCard .data {
-      font-family: "Inconsolata", monospace;
-      font-size: 22px;
-      font-weight: bold;
-      display: flex;
-      flex-direction: row;
-      align-items: flex-end; }
-      .networkStatsWrapper .statCard .data span:first-child {
-        display: flex;
-        flex-direction: row;
-        align-items: flex-end; }
-      .networkStatsWrapper .statCard .data p {
-        margin: 0px; }
-      .networkStatsWrapper .statCard span {
-        line-height: 29px;}
-    .networkStatsWrapper .statCard .subdata {
-      display: flex;
-      flex-direction: row;
-      align-items: flex-end;
-      font-weight: semibold;
-      color: #8997A5; }
-      .networkStatsWrapper .statCard .subdata p {
-        margin: 0px; }
-      .networkStatsWrapper .statCard .subdata span {
-        font-size: 12px;}
-
-.showmore {
-  background: white;
+    line-height: 1;
+    color: #989898;
+    text-align: left;
+}
+.block-info span.timespan {
+    font-family: "Source Sans Pro", sans-serif;
+    line-height: 1;
+    color: #989898;
+    text-align: right;
+}
+.block-rows {
   display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 55px;
-  cursor: pointer;
-  border-radius: 4px; }
-  .showmore p {
-    margin-bottom: 0px;
+  flex-direction: column;
+  height: 90px;
+  padding: 1px;
+}
+.block-rewards, .block-votes, .block-tickets, .block-transactions {
+  flex-basis: 5px;
+  flex-grow: 1;
+  width: 100%;
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: stretch;
+  border-top: 1px solid white;
+}
+.block-rewards {
+    border-top: none;
+}
+.block-rewards > span,
+.block-votes > span,
+.block-tickets > span,
+.block-transactions > span:not(:first-child) {
```

Comment:
The negative psuedo class ":not" isn't widely supported, only works in safari.
https://developer.mozilla.org/en-US/docs/Web/CSS/:not#Browser_compatibility
So we'll have to over-ride to remove the border for first-child with a separate style declaration

Reply:
Noted